packages/backend/src/util/handlers/createExpressSynthqlHandler.test.ts

```diff
@@ -0,0 +1,58 @@
+import { beforeAll, describe, expect, test } from 'vitest';
+import { createExpressSynthqlHandler } from './createExpressSynthqlHandler';
+import { QueryEngine } from '../..';
+import { getMockReq, getMockRes } from 'vitest-mock-express';
```

Comment:
Do you really need an additional library for this?

Reply:
General note: Your default position shouldn't be to add libs if you're in need of something. Dependencies are a dangerous thing. They often bring hidden complexities that you often don't realise until it's too late to remove them.

Reply:
Understood. I did try to create it without, but it meant that we would have to remove the types on `req` and `res` in the handler function, or use partial types, both of which didn't seem ideal.

Reply:
There's a better way: see my comment about http.createServer below. So I think we can get rid of vitest-mock-express